### ISSUE TITLE
nSPV integration

### ISSUE THREAD

**Issue by pbca26** [KomodoPlatform/electrum-komodo#46]

**Comment by pbca26** [comment]
I've investigated several options for nSPV integration. One way would be to do a rewrite of Electrum wallet's network code but it seems to take quite an effort to achieve. A more complex approach is to use a wrapper that's fully compatible with Electrum Server API. ElectrumX is a bit too much for such a "simple" task. I found an old version of Electrum Server which is less complicated (no alt coin support etc) and should suit our needs https://github.com/pbca26/electrum-server. Also, a wrapper will open a path for extra features such as event based methods emulation (since nSPV doesn't have it out of the box) and an ability to switch to newer Electrum wallet code w/o making any changes into nSPV wrapper as it will be likely 99% compatible.

To sum up the idea is to make it work this way Electrum wallet <-> wrapper code <-> nSPV daemon. Wrapper code is going to control and manage nSPV daemon, provide Electrum Server level API for Electrum wallet.
